include/tfm/tfm_read_ranges.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef TFM_READ_RANGES_H__
+#define TFM_READ_RANGES_H__
+
+#include <tfm_ioctl_api.h>
+#include <pm_config.h>
+
+#define FICR_BASE               NRF_FICR_S_BASE
+#define FICR_PUBLIC_ADDR        (FICR_BASE + 0x204)
+#define FICR_PUBLIC_SIZE        0xA1C
+#define FICR_RESTRICTED_ADDR    (FICR_BASE + 0x130)
+#define FICR_RESTRICTED_SIZE    0x8
+
+static const struct tfm_read_service_range ranges[] = {
```

Comment:
Is this an acceptable way of providing this from out-of-tree when we move the rest of the implementation upstream?

Reply:
Not sure I understand this question.

Reply:
If we want to move the .c file upstream with the service implementation, there should ideally be some way to provide the ranges that should be readable from out of tree. 

This is my proposal on how to do it, if we move the .c file upstream, we can still provide this headerfile from out of tree. Also since this is imported directly in the .c file you will get a compiler warning if it is not provided.